boards/phytec/mimx93_phyboard_nash/mimx93_phyboard_nash_a55.dts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022,2024 NXP
+ * Copyright 2024 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "mimx93_phyboard_nash-pinctrl.dtsi"
+
+/ {
+	model = "PHYTEC phyBOARD-Nash i.MX93";
+	compatible = "phytec,mimx93_phyboard_nash";
+
+	chosen {
+		zephyr,console = &lpuart2;
+		zephyr,shell-uart = &lpuart2;
+		zephyr,sram = &sram0;
+	};
+
+	cpus {
+		cpu@0 {
+			status = "disabled";
```

Comment:
why is cpu0 disabled?

Reply:
The imx93 variant on the phyBOARD-Nash has 2 A55 CPUs. I guess the default use-case would be to run Zephyr on one core for specific real-time applications and not to use SMP. NXP usually disabled all but one core in their Cortex-A EVK boards.